lib/phut/open_vswitch.rb

```diff
@@ -33,7 +40,11 @@ def running?
     end
 
     def pid_file
-      "#{Phut.settings['PID_DIR']}/open_vswitch.#{@dpid}.pid"
+      "#{Phut.settings['PID_DIR']}/open_vswitch.#{name}.pid"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.